ArjunaJTS/standalone/README.md

```diff
@@ -68,7 +68,7 @@ Running with JacOrb:
 
 Running with JdkORB:
 
-as above but replace JacORB with JdkORB
+as above but replace jacorb with jdkorb
```

Comment:
The JDK ORB is no longer supported by Narayana and has been removed with work on JBTM-3590 as the minimum Java version is now 11 and given JEP-320 then we no longer should try to work with the JDK ORB.

That's a bit of pre-amble, but the overall point is rather than JDKORB we should be referring to OpenJDK ORB (it's a JBoss fork of that actually: https://github.com/jboss/openjdk-orb)